docs/whats-news/2023/2023-q2.mdx

```diff
@@ -5,7 +5,92 @@ slug: /whats-news/2023-Q2
 sidebar_label: What's Newsletter Q2 2023
 ---
 
-## The `nimbus-cli`
+This is a non-exhaustive list of the Nimbus team's accomplishments in Q2, including some deep dives into a few of the more exciting features that have landed.
+
```

Comment:
Adding blurb for multifeature:
Enables users to to run experiments that target multiple features at once. We modified experiment creation pages to be able to select multiple features via a multiselect at the top of the page (before editing branches). Each feature value is  validated against its respective schema as before. Modified client and API to support multiple features in each branch and ensure existing experiments that are already published continue to use the single feature branch type without impact.

Reply:
epic # https://mozilla-hub.atlassian.net/browse/EXP-1610